### ISSUE TITLE
[meta] Release PR CLA fails

### ISSUE THREAD

**Issue by AndrewJakubowicz** [lit/lit#2711]
# Description

The Release PR CLA check fails:

```
❌ 39586ba PR Opener: @lit-robot
❌ 39586ba Author: @github-actions[bot] <github-actions[bot]​@users.noreply.github.com>
```

Both bots fail CLA checks.

**Comment by augustjk** [comment]
This is curious. Did something change since #2502 was resolved?
Maybe this https://github.com/changesets/action/pull/161 actually makes the commit author be github-actions bot again. lit-robot was working fine before though.

**Comment by augustjk** [comment]
Oh and related #2542

**Comment by augustjk** [comment]
Fixed by #2729 
Passing CLA here: https://github.com/lit/lit/pull/2806
